scrapers/ma/bills.py

```diff
@@ -68,91 +70,55 @@ def scrape(
             else:
                 single_bill_chamber = "upper"
 
-            yield from self.scrape_bill(session, bill_no, single_bill_chamber)
-            return
+            for bill_meta in self.bill_list[single_bill_chamber]:
+                if (
+                    bill_no == bill_meta["DocketNumber"]
+                    or bill_no == bill_meta["BillNumber"]
+                ):
+                    yield from self.scrape_bill(session, bill_meta, single_bill_chamber)
+                    return
```

Comment:
I know the pattern is valid, but this feels weird. Maybe `break` with a comment instead? Or at least explain the `return` here.

Reply:
This is in a nested block, so break won't work -- i don't see a great way to refactor out the nest. I'll drop in a self.info comment through so it's clear what's happening.